meerk40t/gui/mwindow.py

```diff
@@ -35,10 +35,10 @@ def __init__(self, width, height, context, path, parent, *args, style=-1, **kwds
         if self.window_save:
             self.window_context.setting(int, "width", width)
             self.window_context.setting(int, "height", height)
-            if self.window_context.width < 100:
-                self.window_context.width = 100
-            if self.window_context.height < 100:
-                self.window_context.height = 100
+            if self.window_context.width < width:
+                self.window_context.width = width
+            if self.window_context.height < height:
+                self.window_context.height = height
```

Comment:
This would force the windows minimum size to being equal to width/height. That's not correct. If this were the intent a second width/height for min_width, min_height should be passed to this function call.

Reply:
The width and height provided in the __init__ call are the initial sizes of the window. But if you increase these in a PR, the window does not increase in size unless the user manually resets all windows position/size settings. In particular new Preference checkboxes are hidden.

The code already ensures a minimum size on opening if the user has shrunk it to something smaller. All this change does is to change the minimum size on open to something more meaningful without preventing the user from then reducing the size manually if they wish to.

The alternative would be to store the previous default size and compare it to when the window is opened again, and if the default size has changed, to resize it only in that case.